### PR TITLE
Add association

### PR DESCRIPTION
**Pull request by girma336**
#### In this project, i'm going to finish creating the remaining classes for our school library and create the associations between them.

- [ ]  Added new book, rental and classroom classes
- [ ]  Added updated person and student classes

### CODE REVIEW

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @girma336 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error


## Required Changes ♻️
- [ ] Since you have not deployed the project yet, Kindly remove the `live demo` link on your readme file to keep it professional as it is broken
- [ ] Kindly include the correct instructions to install and use the project locally as this is a ruby project hence it does not need a javascript runtime environment
- [ ] Kindly adjust the link to your `MIT` license as it is broken to keep your repo professional

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -90,8 +90,7 @@ After you're finished please remove all the comments and instructions!
 
 In order to run this project you need:
 
-- Node js installed on your computer
-- Npm Node package manager
+- Ruby installed on your computer
 - Terminal
```
Comment (pinkmoon25) [inline README.md:95]: - [ ] As requested by the previous reviewer "_Since you have not deployed the project yet, Kindly remove the live demo link on your readme file to keep it professional as it is broken"_ your live demo link is still broken and giving a 404 error, having a professional documentation is also very important for our projects, please fix it :+1: 
- [ ] As mentioned above and also by the previous reviewer, you also need to fix the getting started section specifically the install one, as it is not a javascript project `npm install` will not do anything replace it with `bundle install`
image

- [ ] Also `npm start ` will throw an error, a user can run this program locally with `ruby rubyfile.rb`
image

- [ ] Same thing for test and deploy you run test with rspec and as it is a console app, there is no deploy\

- [ ] Also mentioned by the previous reviewer "_Kindly adjust the link to your MIT license as it is broken to keep your repo professional" your Mit link is still broken and throwing a 404, add a LICENSE file to the root of your project and fix it 

I hope things are much clear now, please implement the fixes :pray:

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @girma336 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
linters :heavy_check_mark: 
gitflow :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -175,7 +163,7 @@ I would like to thank Microverse for the project.
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./MIT) licensed.
+This project is [MIT](./LICENSE) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (KDlamini) [inline README.md]: [OPTIONAL] Please remove the license suggestion it is meant to guide you on which type of license you can use for your application. It is not meant to be part of the final documentation which is meant for users.

**Review by KDlamini — APPROVED** [review]
### Hi @girma336 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ Professional pull request
✔️ No linter errors

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**